src/menu.js

```diff
@@ -0,0 +1,76 @@
+import item1 from './assets/images/idli.jpg';
+import item2 from './assets/images/dosa.jpg';
+import item3 from './assets/images/vada.jpg';
+import item4 from './assets/images/bisibelebath.jpg';
+
+function itemFactory(name, description, price) {
+  return { name, description, price };
+}
+
+function createDishes() {
+  const menu = [];
+  menu.push(itemFactory('Idli', 'A type of savoury rice cake, originating from the Indian subcontinent, popular as breakfast foods in Southern India and in Sri Lanka. The cakes are made by steaming a batter consisting of fermented black lentils (de-husked) and rice.', 20));
+  menu.push(itemFactory('Dosa', 'A dosa or dose is a thin pancake or crepe, originating from South India, made from a fermented batter predominantly consisting of lentils and rice.', 30));
+  menu.push(itemFactory('Vada', 'A category of savoury fried snacks from India. Different types of vadas can be described variously as fritters, cutlets, doughnuts, or dumplings', 15));
+  menu.push(itemFactory('Bisi Bele Bath', 'A spicy, rice-based dish with origins in the state of Karnataka, India.', 45));
+  return menu;
+}
+
+function collectImages() {
+  const images = [];
+  const dish1 = new Image();
+  const dish2 = new Image();
+  const dish3 = new Image();
+  const dish4 = new Image();
+  dish1.src = item1;
+  dish2.src = item2;
+  dish3.src = item3;
+  dish4.src = item4;
+  images.push(dish1, dish2, dish3, dish4);
+  return images;
+}
+
+function setImages() {
+  const images = collectImages();
+  images.forEach(image => {
+    const index = images.indexOf(image);
+    const placeholder = document.querySelector(`#dish-${index}`);
+    placeholder.appendChild(image);
+  });
+}
```

Comment:
- [ ] Use ES6 syntax, especially the usage of arrow functions instead of the `function` keyword.
- Replace all the rest